src/test/java/ua/lviv/iot/AppTest.java

```diff
@@ -0,0 +1,20 @@
+package ua.lviv.iot;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest 
+{
+    /**
+     * Rigorous Test :-)
```

Comment:
useless comment

Reply:
Thanks, I will remove it. The comment was auto-generated by maven-archetype-quickstart.

Reply:
Fixed ✅